pendant/theme.json

```diff
@@ -1,295 +1,310 @@
 {
-    "version": 2,
-    "templateParts": [
-        {
-            "name": "header",
-            "area": "header"
-        },
```

Comment:
Is this happening because of the spaces to tabs change?

Reply:
Yes.